dynamiq/memory/backend/qdrant.py

```diff
@@ -1,13 +1,14 @@
 import uuid
 
-from qdrant_client.http import models as qdrant_models
-from qdrant_client.http.exceptions import ApiException, UnexpectedResponse
-from qdrant_client.models import FieldCondition, Filter, MatchAny, MatchValue, Range
+from qdrant_client.http.exceptions import UnexpectedResponse
 
 from dynamiq.components.embedders.base import BaseEmbedder
 from dynamiq.connections import Qdrant as QdrantConnection
 from dynamiq.memory.backend.base import MemoryBackend
 from dynamiq.prompts import Message
+from dynamiq.storages.vector.policies import DuplicatePolicy
+from dynamiq.storages.vector.qdrant import QdrantVectorStore
```

Comment:
@olbychos should we also add Weaviate?

Reply:
@vitalii-dynamiq we can add this, but this optional
Let’s proceed with overall implementation memory and stabilize it, and afterwards we can easily in future add new types and so on